app/manifest.json

```diff
@@ -71,7 +71,10 @@
     "46-opensettings": {"description": "LOCATION: Open Settings Page"},
     "47-openextensions": {"description": "LOCATION: Open Extensions Page"},
     "48-openshortcuts": {"description": "LOCATION: Open Keyboard Shortcuts Page"},
-    "49-showlatestdownload": {"description": "DOWNLOADS: Show latest download"}
+    "49-showlatestdownload": {"description": "DOWNLOADS: Show latest download"},
+    "50-capturescreenshot": {"description": "UTILITY: Capture current viewport screenshot"},
+    "51-capturefullsizescreenshot": {"description": "UTILITY: Capture full size screenshot (max height is 16,348px due to browser limitation)"},
+    "52-forcecapturefullsizescreenshot": {"description": "UTILITY: Force capture full size screenshot (when window is not scrollable)"}
```

Comment:
Can you give an example of when the forced capture is necessary? I'm confused about it and worried people wouldn't understand which one to use when.

Reply:
Sorry to let you confused. Normal capturefullsizescreenshot use `document.body.scrollHeight` as full size height, which works well in most cases. But in this kind of site: https://chromedevtools.github.io/devtools-protocol/1-3/Debugger/,  `document.body.scrollHeight === window.innerHeight` (which I described 'window is not scrollable'), it is `body > main > section` scrollable. Force mode is to handle this case.